tests/test_parser/test_complex_outputs.xml

```diff
@@ -244,7 +244,7 @@
             <container classes="cell_output" nb_element="cell_code_output">
                 <container nb_element="mime_bundle">
                     <container mime_type="image/png">
-                        <image candidates="{'*': '_build/jupyter_execute/8c43e5c8cccf697754876b7fec1b0a9b731d7900bb585e775a5fa326b4de8c5a.png'}" uri="_build/jupyter_execute/8c43e5c8cccf697754876b7fec1b0a9b731d7900bb585e775a5fa326b4de8c5a.png">
+                        <image candidates="{'*': '_build/jupyter_execute/8c43e5c8cccf697754876b7fec1b0a9b731d7900bb585e775a5fa326b4de8c5a.png'}" height="400" uri="_build/jupyter_execute/8c43e5c8cccf697754876b7fec1b0a9b731d7900bb585e775a5fa326b4de8c5a.png">
```

Comment:
This one shouldn't get the `height=400` either.

Before this PR:

![imatge](https://github.com/executablebooks/MyST-NB/assets/23738400/08e090b7-53e3-4a60-bd91-f9dc9bdeb8fb)

after this PR:

![imatge](https://github.com/executablebooks/MyST-NB/assets/23738400/e8da67da-aa2f-4b2e-8c8f-5c298dc09a02)

Reply:
For sure!